doc/src/input/log.md

```diff
@@ -0,0 +1,81 @@
+# Logging configuration
+
+Lumol send various logging messages while running a simulation. Some of them are
+informational messages (`charge was set to 1.2 for 132 particles`), others are
+warnings or error message (`infinite energy!`) and some are for debug purposes.
+
+By default, the code print all the informational, warning and error messages to
+the standard terminal output. This allow to run the code and redirect the output
+to a specific file with the usual UNIX way: `lumol input.toml > simulation.log`.
+
+Lumol also offer finer-grain configuration for logging output, for example only
+printing errors and warnings, and redirecting everything else to a file. This
+configuration happens in the `[log]` section of the input file. This section can
+contains either a single output `target`, or multiple `targets`.
```

Comment:
I am really bad at putting the 's' at the third person =/ Thank you for the review!